src/styles/blocks/page.scss

```diff
@@ -0,0 +1,18 @@
+.page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
```

Comment:
Code Style issue: It is a good practice to specify a generic font family as a fallback. In case the first font doesn't load or is not available, the browser can use the generic font family. For example, if you are using a sans-serif font like Roboto, you should end the list with 'sans-serif'.